src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Presentation/CustomCommitCompletion.cs

```diff
@@ -81,6 +82,22 @@ public string GetDescription_TestingOnly()
             return this.CompletionItem.GetDescriptionAsync(CancellationToken.None).WaitAndGetResult(CancellationToken.None).GetFullText();
         }
 
+        public TextRunProperties GetTextRunProperties(TextRunProperties defaultTextRunProperties)
+        {
+            var item = (CompletionItem as DescriptionModifyingCompletionItem)?.CompletionItem ?? CompletionItem;
```

Comment:
The problem with that is that these are all extremely specific WPF types, and we can't have those appear on `CompletionItem`.  Rather than introducing an abstraction over all of the types of text formatting that items might want to do, we settled for this.